agents/fts_aerotech_stage/fts_aerotech_agent.py

```diff
@@ -286,6 +308,7 @@ def make_parser(parser=None):
     pgroup = parser.add_argument_group('Agent Options')
     pgroup.add_argument('--ip-address')
     pgroup.add_argument('--port')
+    pgroup.add_argument('--config_file')
```

Comment:
While there is a mix of '-' and '_' separators in multi-word flags, '-' is more common among our Agents. Can we change this to `--config-file`? (Note, `args.config_file` will still need to be used on line 335.)

Reply:
sounds good, changed.